tests/data/expected/course_contents.csv

```diff
@@ -1,11 +1,11 @@
-section,content_id,activity_name,lesson_page
```

Comment:
Let's remove `visibility` from the expected output, and keep it in the input `data/course_contents.csv` (basically to get code coverage on the fact that the column isn't magically appearing unintentionally).